### PR TITLE
Add support for selinux_permissive module

### PR DESCRIPTION
**Pull request by shamsalmon**
Added support for turning off/on permissive mode for specific domains.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,19 @@ selinux_booleans:
...
+selinux_domains:
+  - { name: 'httpd_t', state: 'permissive' }
+```
+
+```yaml
+selinux_booleans:
+  - { name: 'http', state: 'on' }
+  - { name: 'ssh_sysadm_login', state: 'on', persistent: 'yes' }
+```
+
```
Comment (bachradsusi) [inline README.md]: It looks like copy&paste leftover. selinux_booleans are shown above this chapter.

**Comment by bachradsusi** [comment]
I'd rather use something like `selinux_permissive_domains` so it's clear that it handles just permissive domains.

**Comment by pcahyna** [comment]
> I'd rather use something like `selinux_permissive_domains` so it's clear that it handles just permissive domains.

An alternative would be to keep `selinux_domains` and use it for any other domain attributes that may arise in the future, if such thing is likely to arise.

**Comment by shamsalmon** [comment]
Ill make those changes guys. Thanks!

**Comment by shamsalmon** [comment]
@bachradsusi made some suggested changes. Let me know if you guys would like anything else!

**Comment by bachradsusi** [comment]
I'm fine with this change generally. Could you please provide also a test for the `selinux_domains` ?

**Comment by pcahyna** [comment]
[citest commit:af5a0a64230c78bd9154111a4287064f1f96da59]

**Comment by pcahyna** [comment]
@shamsalmon @bachradsusi what do you think about the proposed `selinux_domains_purge` option? I would like to add it for symmetry with others, unless it makes no sense.

**Comment by bachradsusi** [comment]
>  what do you think about the proposed selinux_domains_purge option?

Given that there's no `semanage permissive -D` yet I'm not sure it's worth to implement it until it's supported directly in `selinux_permissive` module

**Pull request closed without merging**